client/jetpack-connect/signup.js

```diff
@@ -302,7 +314,7 @@ export class JetpackSignup extends Component {
 					'Your account will enable you to start using the features and benefits offered by %(pluginName)s',
 					{
 						args: {
-							pluginName: wooDna.name( translate ),
+							pluginName: wooDna.getServiceName(),
```

Comment:
No more translation for service name?

Reply:
The service name is translated server-side now. Not ideal, because the user could switch their language client-side, but realistically speaking product names aren't usually translated anyway.

Reply:
Yeah, I was wondering which language is used for translation on the server. Is it a store language or account default language or something else? Whatever, this is non-blocker IMO and can be addressed later if required.

Reply:
The site's language is sent to the server in the `site_lang` parameter, but it looks like it's being ignored. I agree that it's not a blocker now, and probably will never be (product names are rarely translated).